src/main/java/bio/terra/janitor/app/configuration/CrlConfiguration.java

```diff
@@ -69,4 +83,28 @@ private static HttpRequestInitializer setHttpResourceManagerTimeout(
       httpRequest.setReadTimeout(3 * 60000); // 3 minutes read timeout
     };
   }
+
+  /** Creates an Azure {@link ComputeManager} client for a given managed resource group. */
+  public ComputeManager buildComputeManager(AzureResourceGroup azureResourceGroup) {
```

Comment:
We might be able to create a `ComputeManager` Bean just like the `Cows` above?  
Then subclasses can inject that bean directly.

Reply:
I thought about that but we need a `ComputeManager` instance for each subscription/resource group. Since Janitor will need to clean up resources across subscriptions, I'm building it on-demand instead of making it a singleton `Bean`.